job-server-api/src/main/scala/spark/jobserver/util/SparkJobUtils.scala

```diff
@@ -82,11 +82,6 @@ object SparkJobUtils {
     // Set the Jetty port to 0 to find a random port
     conf.set("spark.ui.port", "0")
 
-    // Set spark broadcast factory in yarn-client mode
-    if (sparkMaster == "yarn-client") {
```

Comment:
Just curious... is this not needed anymore?

Reply:
The torrent based broadcast factory seems to be the default since spark 1.1.0 and the option was entirely removed in spark 2.0.0.